docs/developing-component.md

```diff
@@ -45,6 +46,91 @@ git clone https://github.com/dapr/components-contrib.git github.com/dapr/compone
 make test
 ```
 
+### Debug unit-tests
+
+Specify the package that you want to test when running the `dlv test`. For example, to debug the `./pkg/actors` tests:
+
+```bash
+dlv test -v ./state/redis
+```
+
+If you want to combine VS Code + dlv for debugging so you can set breakpoints in the IDE create a debug launch configuration as follows:
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+    {
+        "name": "Launch test function",
+        "type": "go",
+        "request": "launch",
+        "mode": "test",
+        "program": "${workspaceFolder}/state/redis",
+        "buildFlags": "-v"
+    },
+]
+}
+```
+
+### Enabling conformance tests for your new component
+
+You will need to update [`tests/conformance/common.go`](https://github.com/dapr/components-contrib/blob/master/tests/conformance/common.go) and `tests/config/<component-type>/tests.yaml` ([bindings components example](https://github.com/dapr/components-contrib/blob/master/tests/config/bindings/tests.yml)) to add your component.
+
+Once conformance tests pass locally also add the test to the GitHub workflow at https://github.com/dapr/components-contrib/blob/master/.github/workflows/conformance.yml
+
+Ensure that any infrastructure required for the test is provisioned via docker-compose ([see this example](https://github.com/dapr/components-contrib/blob/master/.github/infrastructure/docker-compose-kafka.yml)) or work with Dapr maintainers to securely provide credentials to test infrastructure you maintain.
```

Comment:
What is the meaning of `work with Dapr maintainers`? Is that `Dapr containers`?

Reply:
@daixiang0 technically it's the Dapr Technical and Steering Committee (TSC) -- the people who control the github.com/dapr organization because only those people can create new GitHub secrets that can be used in GitHub workflows.

However at the moment there is no permanent TSC elected yet, so I just vaguely called it Dapr maintainers. If somebody pings a maintainer about this the maintainer won't be able to do anything - but they will know to contact the TSC, so that's ok.